ui/src/components/search/services/searchService.js

```diff
@@ -48,13 +48,15 @@ export const searchModels = async (inputValue, apiFetcher) => {
     op: 'or',
     content: [
       {
-        op: 'filter',
-        content: { value: `*${inputValue.toLowerCase()}*`, fields: ['autocomplete'] },
+        op: 'in',
+        content: { value: `*${inputValue.toLowerCase()}*`, fieldName: 'autocomplete' },
```

Comment:
Just a couple questions about this change:
1. What is the functional difference between the `'filter'` and `'in'` SQON operations?
2. Is it correct to replace `fields: []` with `fieldName: string` here?

As long as the search by model name input is working as expected, this _should_ be fine. But I was just curious to know if these changes could have any unintended consequences.

Reply:
Discussed on Slack, fieldName and field operation filter are updated Arranger 3 Sqon conventions. Model search autocomplete & onClick sqon updates are working in this branch.